package.json

```diff
@@ -8,7 +8,10 @@
     "start": "astro dev",
     "build": "astro check && astro build",
     "preview": "astro preview",
-    "astro": "astro"
+    "astro": "astro",
+    "format": "pnpm run format:code && pnpm run format:imports",
+    "format:code": "biome format ./ --write",
+    "format:imports": "biome check --apply ."
```

Comment:
Do we need this one?

Reply:
do you want me to remove it or are you asking? I'm not sure what's needed really, but there's also `pnpm biome check --apply <files>` that works as both `format` and `check`

https://biomejs.dev/guides/getting-started/#usage

Reply:
I like to have those commands https://github.com/florian-lefebvre/astro-integration-kit/blob/main/package.json#L14-L15, one to check if needed and the other to just do everything

Reply:
I personally don't understand why combining linting and formating has any benefits, but I won't die on that hill. So happy with whatever Florian said.

Reply:
done https://github.com/astrolicious/astro-tips.dev/pull/80/commits/4698d07c06a4ae5894eecdcf1885cf3cac6365e3